source/primer/devops/continuous_integration.rst

```diff
@@ -0,0 +1,117 @@
+Continuous Integration
+=====================
+
+.. admonition:: :fa:`graduation-cap` Learning Objectives
+
+    .. include:: objectives/continuous_integration.rst
+
+In the context of Tudat, Continuous Integration is used to streamline the development process.
+Thanks to it, developers need not worry about every step required to build and release the conda packages and documentation.
+Instead, they can either wait for nightly releases that are made automatically, or add specific tags in a commit message and let everything else run hidden trough :ref:`GitHub<GitHub actions>`, :ref:`Azure<Azure>`, :ref:`Anaconda<Anaconda>`, and :ref:`Readthedocs<Readthedocs>`.
+
+GitHub actions
+--------------
+.. image:: https://github.com/tudat-team/.github/actions/workflows/webservices.yml/badge.svg
+    :target: https://github.com/tudat-team/.github/actions/workflows/webservices.yml
+
+GitHub actions are integrated in Tudat development process to ease and automate the build and release of conda packages.
+
+Most importantly, the `webservices-dispatch-action repository <https://github.com/tudat-team/webservices-dispatch-action>`_ contains code that is used to trigger a build of a conda package.
+More information on how this action works is given in that repo's `README <https://github.com/tudat-team/webservices-dispatch-action/blob/main/README.md>`_.
+
+In essence, once triggered, this action can do two distinct things:
+
+* Bump the version of the project and project feedstock repositories.
+* Re-render the feedstock repository, then bump the version as above.
+
+It is worth noting that this action makes the distinction between ``main``/``master`` repositories, and ``dev``/``develop`` repositories, so that the proper versions are bumped, and the correct build triggered.
+
+In both cases, if changes are caused by the action, they are automatically pushed first to the project repository, then to the feedstock repository.
+As a reminder, pushing to the feedstock repository will then trigger a build of the conda package trough Azure (as discussed :ref:`below<Azure>`).
+
+Last but not least, this action is triggered automatically either every 24 hours (nightly), or by adding a ``[CI]`` or ``[rerender]`` tag in a commit to a project repository.
```

Comment:
These tags have to be at the beginning of the commit message right?

Reply:
They don't, and the capitalisation also does not matter. This can easily be changed if needed (the tags are detected using [these lines of code](https://github.com/tudat-team/webservices-dispatch-action/blob/a01beb5c4f78ac454753ba0473837b52f05b0c4d/util.py#L111-L120)). I updated the docs with this information in ffe42e84998cfe41ca2e852c10068aeaab01d56d.